Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst

```diff
@@ -190,16 +190,18 @@ Environment Variables
 
 Some specific flow behaviour can also be configured with a couple of environment variables.
 
-============================	==================================================
-Variable						Description
-============================	==================================================
-FLOW_ROOTPATH					Can be used to override the path to the Flow root.
-FLOW_CONTEXT					Use to set the flow context (see above).
-FLOW_PATH_TEMPORARY_BASE		Can be used to set a path for temporary data.
-FLOW_LOCKHOLDINGPAGE			Use to specify the html page shown when the site is locked.
-								This is relative to the Packages directory.
-FLOW_ONLY_COMPOSER_LOADER		Set to true (1) to only use composer autoloader.
-============================	==================================================
+============================= ==================================================
+Variable                      Description
+============================= ==================================================
+``FLOW_ROOTPATH``             Can be used to override the path to the Flow root.
+``FLOW_CONTEXT``              Use to set the flow context (see above).
+``FLOW_PATH_TEMPORARY_BASE``  Can be used to set a path for temporary data.
+``FLOW_LOCKHOLDINGPAGE``      Use to specify the html page shown when the site is
+                              locked. This is relative to the Packages directory.
+                              Can be given as ``FLOW_LOCKHOLDINGPAGE``, too.
+                              That is deprecated as of Flow 8.0.
```

Comment:
Shall we mention deprecation in Flow 8 in the 6.3+ docs? Feels weird, but I guess it doesn't hurt and makes things more transparent, so...